src/main/java/duke/tasksmanager/Tasks.java

```diff
@@ -0,0 +1,32 @@
+package duke.tasksmanager;
+
+public class Tasks {
+
+    public String description;
+    public boolean isDone;
+    public String typeOfTask;
+    public String date;
+
+    public Tasks(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
```

Comment:
Good job in keeping the methods  [simple](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#practice-kissing) here.